plugins/backfill/storage/transformerExporter.go

```diff
@@ -17,8 +17,8 @@ var Exporter exporter
 func (e exporter) Export() ([]event.TransformerInitializer, []storage.TransformerInitializer, []interface1.ContractTransformerInitializer) {
 	return []event.TransformerInitializer{},
 		[]storage.TransformerInitializer{
-				flip_univ2aaveeth_a_v1_2_7.StorageTransformerInitializer,
-				flip_univ2wbtcdai_a_v1_2_7.StorageTransformerInitializer,
+			flip_univ2aaveeth_a_v1_2_7.StorageTransformerInitializer,
```

Comment:
Not merge blocking but we can remove this right?

Reply:
Good call—this needs another build anyways, so I'll remove them.